__device-tests__/helpers/caps.js

```diff
@@ -13,8 +13,8 @@ exports.ios12 = {
 exports.android8 = {
 	browserName: '',
 	platformName: 'Android',
-	platformVersion: '8.0',
-	deviceName: 'Android Emulator',
+	platformVersion: '9.0',
+	deviceName: 'Google Pixel 3 GoogleAPI Emulator',
```

Comment:
Right! So this is ignored locally because https://github.com/wordpress-mobile/gutenberg-mobile/blob/add/tests-add-block-insertion/__device-tests__/helpers/utils.js#L85, this is to change was mainly a change to the device on CI to a more commonly used one, you can look at one of the video runs to see that change, it's not a super important detail so if you think we need to rever that I have no problem with that.

Reply:
No need to revert, I was just wondering. Thanks for explaining!